src/app/component/about-page/about-page.component.html

```diff
@@ -0,0 +1,162 @@
+<div class="container">
```

Comment:
general: empty lines can be used in some places for better separation of the code structure

Reply:
general: replace magic numbers from class names and IDs. Rename appropriate names in stylesheet

Reply:
Fixed